bnd.bnd

```diff
@@ -0,0 +1 @@
+Sling-Nodetypes: SLING-INF/nodetypes/resourcemerger.cnd
```

Comment:
Should this have a new line at the end? If it doesn't matter I would put one maybe (minor nitpick anyways).